src/main/java/com/ranull/graves/util/InventoryUtil.java

```diff
@@ -132,6 +137,10 @@ public static boolean isBoots(ItemStack itemStack) {
                         "CHAINMAIL_BOOTS");
     }
 
+    public static boolean isOffhandItem(ItemStack itemStack) {
```

Comment:
Is it always this? Can't you also hold something else in the off-hand?

I'm thinking about the dualwield plugin I forked. So then you'd be able to hold an axe or sword in your offhand.

Isn't there a better way to do this? I mean: maybe it's possible to store (as underneath it's using SQLite) the order of items? And as such, also store armor/hand/offhand data?

Do you think it's possible to do it that way?

Reply:
If you allow me to write to your branch, I will have a try today.